lib/common/actions/index.js

```diff
@@ -42,43 +57,60 @@ export function secureFetch (url: string, method: string = 'get', payload?: any,
     return fetch(url, {method, headers, body})
       // Catch basic error during fetch
       .catch(err => {
-        const message = `Error making request: (${err})`
-        consoleError(message)
+        const message = getErrorMessage(status, method, url)
+        console.error(message, err)
         return dispatch(setErrorMessage({message}))
       })
-      .then(res => {
-        // if raw response is requested
-        if (raw) return res
-        let action, message
-        // check for errors
-        const {status} = res
-        if (status >= 500) {
-          action = 'RELOAD'
-          message = `Network error!\n\n(${method} request on ${url})`
-          consoleError(message)
-          dispatch(setErrorMessage({message, action}))
-          return null
-        } else if (status >= 400) {
-          action = status === 401
-            ? 'LOG_IN'
-            : actionOnFail
-          res.json()
-            .then(json => {
-              const {detail, message} = json
-              const unknown = `Unknown (${status}) error occurred while making request`
```

Comment:
Replaced this error with less vague `Error (${status}) making ${method} request to ${url}`